api/src/routes/register.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const config = require('config');
+const jwt = require('jsonwebtoken');
+const router = express.Router();
+const secret = config.get('secret');
+const Joi = require('@hapi/joi');
+
+const userSchema = mongoose.Schema({
+    firstName: String,
+    lastName: String,
+    email: String,
+    phone: Number,
```

Comment:
As for me it is better to store the phone in string type, because user can enter any kind of phone like `056 47 474 434` or `+44-545-545` and it will not be valid for type: Number.